packages/extension-text-align/src/text-align.ts

```diff
@@ -20,7 +20,7 @@ export interface TextAlignOptions {
    * @default 'left'
    * @example 'center'
    */
-  defaultAlignment: string,
+  defaultAlignment: string | null,
```

Comment:
Should we be stricter on the types here, can only be left, right & center, correct?

Same for alignments too